.env.rc1

```diff
@@ -73,7 +73,7 @@ GSTORAGE_DATA_BUCKET=celo-chain-backup/mainnet
 # A list of every valid context. Must start with one of: gcp,aws,azure
 # each context should have its own environment variables, generally of the form
 # <context in all caps and underscores>_*
-CONTEXTS=azure-oracle-westus,azure-oracle-westeurope,azure-oracle-eastus2,gcp-forno-us-west1,gcp-forno-us-east1,gcp-forno-asia-east1,gcp-forno-europe-west1,gcp-forno-southamerica-east1,azure-komenci-southbr,azure-komenci-sea
+CONTEXTS=azure-oracle-westus,azure-oracle-westeurope,azure-oracle-eastus2,gcp-forno-us-west1,gcp-forno-us-east1,gcp-forno-asia-east1,gcp-forno-europe-west1,gcp-forno-southamerica-east1,azure-komenci-southbr,azure-komenci-sea,azure-odis-westus2-a,azure-odis-eastasia-a,azure-odis-westeurope-a,azure-odis-brazilsouth-a
```

Comment:
I assume `-a` is added so that we can have more than one cluster per region. Given that we know now that we can deploy multiple signer replicas to a single cluster, should we drop this convention?

Reply:
We certainly can, though kevin and I found that having a naming convention where you can differentiate between old and new clusters is sometimes useful for situations when you have multiple versions of the "same" cluster deployed at once. We've been in this situation a lot on alfajores and it does get confusing